src/store/modules/job/actions.ts

```diff
@@ -765,6 +765,125 @@ const actions: ActionTree<JobState, RootState> = {
   },
   clearPipelineFilters({ commit }) {
     commit(types.JOB_PIPELINE_FILTERS_CLEARED);
+  },
+  addToBulkScheduler({ commit, state }, payload) {
+    const enums = {
+      ...JSON.parse(process.env?.VUE_APP_ODR_JOB_ENUMS as string) as any,
+      ...JSON.parse(process.env?.VUE_APP_PRODR_JOB_ENUMS as string) as any,
+      ...JSON.parse(process.env?.VUE_APP_PRD_JOB_ENUMS as string) as any,
+      ...JSON.parse(process.env?.VUE_APP_INV_JOB_ENUMS as string) as any,
+      ...JSON.parse(process.env?.VUE_APP_INITIAL_JOB_ENUMS as string) as any,
+    }
+    
+    let appJobId = '', freqType = '';
+    Object.keys(enums).forEach((jobId: string) => {
+      if(enums[jobId] === payload.jobId) appJobId = jobId;
+    })
+
+    const jobFrequencyType = JSON.parse(process.env?.VUE_APP_JOB_FREQUENCY_TYPE as string);
+    Object.keys(jobFrequencyType).forEach((jobId: string) => {
+      if(jobId === appJobId) freqType = jobFrequencyType[jobId];
+    })
+
+    if(freqType) {
+      payload.freqType = freqType;
+      if(freqType === 'slow') showToast(translate("This job has slow frequency type, hence, maxmimum frequency will be set automatically"))
```

Comment:
We should change the frequency for job when freq type is slow.

Reply:
Updated sir